src/vicsetup/libvicsetup/argon2/.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: c
```

Comment:
We are using Azure DevOps for CI, do we need this?

Reply:
See also https://github.com/lsds/sgx-lkl/issues/579. argon2 should be a submodule.

Reply:
@davidchisnall: I just copied argon2 from the argon2 distro. Maik is right, I should have made argon2 a submodule.